recipes-core/disk-encryption/init

```diff
@@ -10,12 +10,17 @@ set -eux -o pipefail
 # Short-Description:    Start and stop the disk-encryption daemon
 ### END INIT INFO
 
+# Source environment variables if file exists
+[ -r /etc/default/disk-encryption ] && . /etc/default/disk-encryption
+
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 NAME=disk-encryption
 DESC="Disk Encryption"
 KEYSIZE=64
 LOGFILE=/var/log/disk-encryption.log
 TPM_NV_INDEX=0x1500016
+# Default LUN number, can be overridden by environment variable
+TARGET_LUN="${TARGET_LUN:-10}"
```

Comment:
This way we have two places for defining the default: here and `/etc/default/disk-encryption`. Either don't create the file if TARGET_LUN is not passed, or remove the TARGET_LUN variable from here

Reply:
we can remove it from here. I just added this here extra but we can remove it.